src/components/FilterList/SearchResults/hooks.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+
+import { useFilterListProvider } from '../hooks';
+
+export function useSearch<T>(
```

Comment:
```suggestion
export function useSearchResults<T>(
```
?  Makes it clearer what the hook does